classes/PodsUI.php

```diff
@@ -4653,9 +4655,13 @@ public function pagination( $header = false ) {
 
 		if ( false !== $this->pagination ) {
 			if ( 1 < $total_pages ) {
+				$first_link = esc_url( $request_uri . ( $append ? '&' : '?' ) . 'pg' . $this->num . '=1' );
+				$prev_link  = esc_url( $request_uri . ( $append ? '&' : '?' ) . 'pg' . $this->num . '=' . max( $this->page - 1, 1 ) );
```

Comment:
:no_entry_sign: Inline IF statements are not allowed

<sub>hash: 86907a983e574eee0da23e9906b89ae4</sub>